pkg/lwip/contrib/lwip.c

```diff
@@ -97,7 +97,7 @@ static socket_zep_t socket_zep_devs[LWIP_NETIF_NUMOF];
 
 #ifdef MODULE_ESP_WIFI
 extern esp_wifi_netdev_t _esp_wifi_dev;
-extern void esp_wifi_setup (esp_wifi_netdev_t* dev);
+extern void esp_wifi_setup(esp_wifi_netdev_t *dev);
```

Comment:
Please put this unrelated style fix in a separate commit (it is okay to remain in this PR though)